core/src/main/java/org/elasticsearch/indices/recovery/PeerRecoveryTargetService.java

```diff
@@ -366,7 +367,7 @@ public static long getStartingSeqNo(final RecoveryTarget recoveryTarget) {
             } else {
                 return SequenceNumbers.UNASSIGNED_SEQ_NO;
             }
-        } catch (final IOException e) {
+        } catch (final IOException|TranslogCorruptedException e) {
```

Comment:
hmm is this a bug?

Reply:
it's a left over from my previous change. Will remove.

Reply:
thx